lib/app/routes/static.rb

```diff
@@ -17,6 +17,18 @@ class Static < Core
         erb :"site/about", locals: {active_languages: active_languages, upcoming_languages: upcoming_languages, planned_languages: planned_languages}
       end
 
+      get '/submitter-workflow' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.